.github/workflows/unit_test_matrix.yml

```diff
@@ -0,0 +1,76 @@
+name: Unit Tests
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  unit_tests:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      matrix:
+        python_version: ${{ matrix.python_versions }}
+        tf_version: ${{ matrix.tf_versions }}
```

Comment:
Looks like these variables are unused. An example of setting envvars: https://github.com/onnx/onnx/blob/093a8d335a66ea136eb1f16b3a1ce6237ee353ab/.github/workflows/main.yml#L118-L122